src/index.tsx

```diff
@@ -190,6 +190,10 @@ export default class SimpleMDEEditor extends React.PureComponent<
   addExtraKeys = () => {
     // https://codemirror.net/doc/manual.html#option_extraKeys
     if (this.props.extraKeys) {
+      Object.assign(
+        this.props.extraKeys,
+        this.simpleMde!.codemirror.getOption("extraKeys")
+      );
       this.simpleMde!.codemirror.setOption("extraKeys", this.props.extraKeys);
```

Comment:
```suggestion
      this.simpleMde.codemirror.setOption(
        "extraKeys",
        Object.assign(
          {},
          this.props.extraKeys,
          this.simpleMde.codemirror.getOption("extraKeys")
        )
      );
```
I would prefer not to mutate original props :) Unsure it's completely safe to do :)

Reply:
Yeah, you're right. Thanks for your suggestion.